.github/workflows/nightly-upstream-snapshot-build.yml

```diff
@@ -43,6 +45,17 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN_NIGHTLY }}
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
```

Comment:
region does not match the region of the ecr repository. can you check if it is safe to change the default aws region in this workflow only?